markdowntext/src/main/java/dev/jeziellago/compose/markdowntext/CustomTextView.kt

```diff
@@ -0,0 +1,46 @@
+package dev.jeziellago.compose.markdowntext
+
+import android.content.Context
+import android.text.Spannable
+import android.text.style.ClickableSpan
+import android.view.MotionEvent
+import androidx.appcompat.widget.AppCompatTextView
+
+class CustomTextView(context: Context?) : AppCompatTextView(context!!) {
```

Comment:
The `Context` here should be required.
```suggestion
class CustomTextView(context: Context) : AppCompatTextView(context) {
```

Reply:
didn't notice that😢